README.md

```diff
@@ -4,6 +4,7 @@
 ![GitHub branch check runs](https://img.shields.io/github/check-runs/kubewarden/sbomscanner/main)
 ![GitHub contributors](https://img.shields.io/github/contributors/kubewarden/sbomscanner)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubewarden/sbomscanner)](https://goreportcard.com/report/github.com/kubewarden/sbomscanner)
+[![Go Reference](https://pkg.go.dev/badge/github.com/kubewarden/sbomscanner.svg)](https://pkg.go.dev/github.com/kubewarden/sbomscanner)
```

Comment:
maybe go docs?

Reply:
I don't have a strong opinion about that. I'm fine going with the change Fabrizio proposed

Reply:
I see all the projects use Go reference, and the icon shows that too, so just ignore my comment.